android/fastlane/Fastfile

```diff
@@ -6,12 +6,12 @@ CREDENTIALS_GIT_REPOSITORY_URL = ENV['CREDENTIALS_GIT_REPOSITORY_URL']
 CREDENTIALS_DIRECTORY_PATH = ENV['CREDENTIALS_DIRECTORY_PATH']
 CREDENTIALS_KEYSTORE_PATH = ENV['CREDENTIALS_KEYSTORE_PATH']
 
-KEYSTORE_PATH = ENV['KEYSTORE_PATH'] || 'test.keystore'
-KEYSTORE_KEY_ALIAS = ENV['KEYSTORE_KEY_ALIAS'] || 'test'
-KEYSTORE_PASSWORD = ENV['KEYSTORE_PASSWORD'] || '123456'
-KEYSTORE_KEY_PASSWORD = ENV['KEYSTORE_KEY_PASSWORD'] || '123456'
+KEYSTORE_PATH = ENV['KEYSTORE_PATH']
+KEYSTORE_KEY_ALIAS = ENV['KEYSTORE_KEY_ALIAS']
+KEYSTORE_PASSWORD = ENV['KEYSTORE_PASSWORD']
+KEYSTORE_KEY_PASSWORD = ENV['KEYSTORE_KEY_PASSWORD']
 
-desc "Prepare the keystore"
+desc "Downloads and decrypts the JKS"
```

Comment:
I think this is a different naming. Comments normally don't use third person in our project.